core/modules/filter/js/filter.js

```diff
@@ -104,6 +104,39 @@ Backdrop.filterEditorDetach = function(field, format, trigger) {
   }
 };
 
+/**
+ * Provides summary text for the "Editor details" fieldset, under each CKEditor
+ * instance.
+ */
+Backdrop.behaviors.filterFieldsetSummaries = {
+  attach: function (context) {
+    $(context).find('fieldset.filter-wrapper').backdropSetSummary(function (element) {
+      var $element = $(element);
+      var $summary = '';
+      // When multiple text formats are available to the current user, these are
+      // being rendered as a select list. In that case, we grab the
+      // human-readable name of the currently selected format from the
+      // respective option in the select element.
+      var $selected_editor_name = $element.find('select.filterEditors-processed :selected').text();
+      // When the current user has access to only a single text format, there is
+      // no select element being rendered from which to get the human name of
+      // the text format. In that case, we grab the human name from the
+      // data-available-editor HTML attribute, which has been added to the
+      // respective hidden input element instead.
+      var $single_avaliable_editor_name = $element.find('input.filterEditors-processed').attr('data-available-editor');
+
+      if ($selected_editor_name.length !== 0) {
+        $summary = $selected_editor_name;
+      }
+      else if ($single_avaliable_editor_name !== 0) {
+        $summary = $single_avaliable_editor_name;
+      }
```

Comment:
I think we don't need so much description here, we can check for the existence of each element via its length() (https://stackoverflow.com/a/587408/4168619) and we can move the actual getting of the name part to the if statement (as I think this reads a bit nicer).

```suggestion
      // Look for a select list of text formats.
      var $select_list = $element.find('select.filterEditors-processed :selected');
      // Otherwise look for a hidden input element.
      var $input_element = $element.find('input.filterEditors-processed');

      if ($select_list.length) {
        $summary = $select_list.text();
      }
      else if ($input_element.length) {
        $summary = $input_element.attr('data-format-name');
      }
```

Reply:
Excellent suggestion 👍 ...I would still want us to include "when the current user has access to only a single text format" ...it took me a bit to figure that out 😅